README.md

```diff
@@ -13,7 +13,7 @@
 
 <a href="https://www.kics.io" title="www.kics.io"><img src="docs/img/button_www-kics-io.png" align="right"></a>
 
-**KICS** (pronounced as 'kick-s') or **Kicscan** is an open source solution for static code analysis of Infrastructure as Code.
+**KICS** (pronounced as 'kick-s') is an open source solution for static code analysis of Infrastructure as Code.
```

Comment:
I'd remove this as well. no one uses this pronounciation :)

Reply:
I agree :)